de/_contact/index.html

```diff
@@ -20,12 +20,12 @@
           <label for="email">您的邮箱</label>
```

Comment:
we should have no longer a contact page. we're using zendesk instead. Can you delete? No link should be left to this page.